### DISCUSSION TITLE
Capture play details and copy to clip board

### DISCUSSION THREAD

**Discussion by RealCrayfish** [ppy/osu#19067] [Ideas]
I think that it would be cool to have a "score capture" function that allows you to copy/save an image of a play's details making it easier to share. Or even maybe being able to copy a link to the the play's details on the website.

An example is something like this:
![image](https://user-images.githubusercontent.com/77017028/178140664-f70c6f04-eb0a-408f-9b90-1de3deea02a4.png)

**Comment by peppy** [comment]
For now you can use F12 to capture a screenshot.
